resources/js/features/games/hooks/useAllMetaRowElements.ts

```diff
@@ -0,0 +1,279 @@
+        altLabels: ['Subgenre'],
+        fallbackValue: game.genre,
+      }),
+    [allGameHubs, game.genre],
+  );
+
+  const languageRowElements = useMemo(
+    () =>
+      buildMetaRowElements({
+        hubs: allGameHubs,
+        hubTitleIncludes: ['Meta - Language -', 'Meta - Language Patch -'],
+        primaryLabel: 'Meta - Language',
+        altLabels: ['Meta - Language Patch'],
+        altLabelsLast: true,
+        markAltLabels: true,
+      }),
+    [allGameHubs],
+  );
+
+  const themeRowElements = useMemo(
+    () =>
+      buildMetaRowElements({
+        hubs: allGameHubs,
+        hubTitleIncludes: ['Theme -'],
+        primaryLabel: 'Theme',
+      }),
+    [allGameHubs],
+  );
+
+  const perspectiveRowElements = useMemo(
+    () =>
+      buildMetaRowElements({
+        hubs: allGameHubs,
+        hubTitleIncludes: ['Perspective -'],
+        primaryLabel: 'Perspective',
+      }),
+    [allGameHubs],
+  );
+
+  const featureRowElements = useMemo(
+    () =>
+      buildMetaRowElements({
+        hubs: allGameHubs,
+        hubTitleIncludes: ['Feature -', 'Game Mechanic -'],
+        primaryLabel: 'Feature',
+        altLabels: ['Game Mechanic'],
+        altLabelsLast: false,
+      }),
+    [allGameHubs],
+  );
+
+  const creditRowElements = useMemo(
+    () =>
+      buildMetaRowElements({
+        hubs: allGameHubs,
+        hubTitleIncludes: ['Credits -'],
+        primaryLabel: 'Credits',
+      }),
+    [allGameHubs],
+  );
+
+  const technicalRowElements = useMemo(
+    () =>
+      buildMetaRowElements({
+        hubs: allGameHubs,
+        hubTitleIncludes: ['Technical -'],
+        primaryLabel: 'Technical',
+      }),
+    [allGameHubs],
+  );
+
+  const miscRowElements = useMemo(
+    () =>
+      buildMetaRowElements({
+        hubs: allGameHubs,
+        hubTitleIncludes: ['Misc. -'],
+        primaryLabel: 'Misc.',
+      }),
+    [allGameHubs],
+  );
+
+  const protagonistRowElements = useMemo(
+    () =>
+      buildMetaRowElements({
+        hubs: allGameHubs,
+        hubTitleIncludes: ['Protagonist -'],
+        primaryLabel: 'Protagonist',
+      }),
+    [allGameHubs],
+  );
+
+  const settingRowElements = useMemo(
+    () =>
+      buildMetaRowElements({
+        hubs: allGameHubs,
+        hubTitleIncludes: ['Setting -'],
+        primaryLabel: 'Setting',
+      }),
+    [allGameHubs],
+  );
+
+  const regionalRowElements = useMemo(
+    () =>
+      buildMetaRowElements({
+        hubs: allGameHubs,
+        hubTitleIncludes: ['Regional -'],
+        primaryLabel: 'Regional',
+      }),
+    [allGameHubs],
+  );
+
+  const raFeatureRowElements = useMemo(
+    () =>
+      buildMetaRowElements({
+        hubs: allGameHubs,
+        hubTitleIncludes: [
+          'Meta -',
+          'Rollout Sets -',
+          'AotW -',
+          'RANews -',
+          'RA Awards -',
+          'Custom Awards -',
+          'Console Wars',
+          'DevJam',
+          'Challenge League',
```

Comment:
While the "Meta - " hubs do seem to be "RA Features", the rest of these aren't. It feels weird to see "AotW" as a feature:
![image](https://github.com/user-attachments/assets/ecfca99a-4759-484a-b427-457f85ecdf78)

Maybe you could split most of these off into a "Featured in" row?

Reply:
I decided to go with a dedicated row just for event hubs in latest:
![Screenshot 2025-05-18 at 4 59 14 PM](https://github.com/user-attachments/assets/52c49272-9c25-423b-98e7-90d650c15fcd)